### ISSUE TITLE
object-curly-spacing missing arraysInObjects exception & schema

### ISSUE THREAD

**Issue by xjamundx** [eslint/eslint#2752]
I forgot to add the `arraysInObjects` exceptions to `object-curly-spacing`. 

Here is how it should work:

When `"arraysInObjects"` is set to `false`, the following patterns are considered warnings:

```js
var obj = { "foo": [ 1, 2 ] };
var obj = { "foo": [ "baz", "bar" ] };
```

The following patterns are not warnings:

```js
var obj = { "foo": [ 1, 2 ]};
var obj = { "foo": [ "baz", "bar" ]};
```

It's also missing a schema.
